### ISSUE TITLE
Implement RPC method Filecoin.F3ListParticipants

### ISSUE THREAD

**Issue by hanabi1224** [ChainSafe/forest#4909]
# Summary

Blocked on #4861  (The response of the method contains lease info that is not yet implemented in Forest)

### F3ListParticipants
F3ListParticipants returns the list of miners that are currently participating in F3 via this node.

Perms: read

Inputs: `null`

Response:
```json
[
  {
    "MinerID": 42,
    "FromInstance": 42,
    "ValidityTerm": 42
  }
]
```

# Completion Criteria

- [x] Port https://github.com/filecoin-project/lotus/pull/12608
- [ ] Update method implementation to align with Lotus

# Additional Links & Resources

**Comment by masih** [comment]
Please note the API was changed after PR review; see: https://github.com/filecoin-project/lotus/pull/12608

**Comment by hanabi1224** [comment]
@masih Thanks for the heads up! Will follow up

**Comment by hanabi1224** [comment]
Unblocked and implemented in #4920
